draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -352,7 +350,7 @@ caches from storing and reusing a response.
 
 Origin servers SHOULD NOT fold multiple Set-Cookie header fields into a single
 header field. The usual mechanism for folding HTTP headers fields (i.e., as
-defined in Section 3.2.2 of {{RFC7230}}) might change the semantics of the Set-Cookie header
+defined in Section 5.4.1 of {{HTTPSEM}}) might change the semantics of the Set-Cookie header
```

Comment:
```suggestion
defined in Section 5.3 of {{HTTPSEM}}) might change the semantics of the Set-Cookie header
```

Reply:
I'm going to merge this one as well since I think it was missed. 5.4.1 doesn't exist in the most recent draft, and the text from the previous draft (https://tools.ietf.org/html/draft-ietf-httpbis-semantics-12#section-5.4.1) is now mostly in 5.3 (https://tools.ietf.org/html/draft-ietf-httpbis-semantics-15#section-5.3).